### PR TITLE
feat: add GPP stub

### PR DESCRIPTION
**Pull request by pierrevano**

### CODE REVIEW

**Comment by sevriugin** [comment]
LGTM, is it stub code from the latest version of iabgpp-es ?

File: scripts/gpp_stub.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+              supportedAPIs: [
+                "2:tcfeuv2",
+                "5:tcfcav1",
+                "6:uspv1",
+                "7:usnatv1",
+                "8:uscav1",
+                "9:usvav1",
+                "10:uscov1",
+                "11:usutv1",
+                "12:usctv1",
```
Comment (ermurri) [inline scripts/gpp_stub.js]: I think we're going to want to match this list: https://github.com/IABTechLab/iabgpp-es/blob/3.1.4/modules/cmpapi/src/encoder/section/Sections.ts#L15-L30

File: scripts/gpp_stub.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+              "2:tcfeuv2",
+              "5:tcfcav1",
+              "6:uspv1",
+              "7:usnatv1",
+              "8:uscav1",
+              "9:usvav1",
+              "10:uscov1",
+              "11:usutv1",
+              "12:usctv1",
+            ],
```
Comment (ermurri) [inline scripts/gpp_stub.js]: same thing here, I think: https://github.com/IABTechLab/iabgpp-es/blob/3.1.4/modules/cmpapi/src/encoder/section/Sections.ts#L15-L30

**Comment by ermurri** [comment]
> LGTM, is it stub code from the latest version of iabgpp-es ?

Hmmm..., if it's taken from this: https://github.com/IABTechLab/iabgpp-es/blob/3.1.4/modules/stub/src/stub.js, then perhaps my other comments are invalid, it seems the stub has fewer sections as part of the `supportedAPIs` than as defined in the encoder sections.  What do you think @sevriugin ?

**Comment by ermurri** [comment]
@sevriugin we can merge this for now and make subsequent changes as needed, if you think this looks good.

**Comment by pierrevano** [comment]
@sevriugin @ermurri I have updated the stub and used directly the one from: https://github.com/IABTechLab/iabgpp-es/blob/master/modules/stub/src/stub.js

**Comment by sevriugin** [comment]
LGTM

**Comment by sevriugin** [comment]
LGTM

**Review by sevriugin — APPROVED** [review]

**Pull request merged**